src/options/quantifiers_options

```diff
@@ -294,10 +294,13 @@ option sygusCRefEval --sygus-cref-eval bool :default true
   direct evaluation of refinement lemmas for conflict analysis
 option sygusCRefEvalMinExp --sygus-cref-eval-min-exp bool :default true
   use min explain for direct evaluation of refinement lemmas for conflict analysis
-  
+
+option checkSynthSol --check-synth-sol bool :default false
```

Comment:
This option should be moved to src/options/smt_options, since this is a feature of the SmtEngine